pandas/core/indexing.py

```diff
@@ -656,6 +578,65 @@ def setter(item, v):
             self.obj._data = self.obj._data.setitem(indexer=indexer, value=value)
             self.obj._maybe_update_cacher(clear=True)
 
+    def _setitem_with_indexer_missing(self, indexer, value):
```

Comment:
Can you annotate this?

Reply:
itd just be Any for both of em.  Is that materially better than leaving it blank?  In general I prefer to leave it blank rather than put a too-loose type that I'm not sure about